docs/website/docs/building-from-source/ios.md

```diff
@@ -0,0 +1,130 @@
+# Cross-compilation for iOS
+
+Cross-compilation for iOS consists of the two steps below.
+
+* On the macOS host, build the IREE compiler.  We can run it to create
+  IREE modules.
+* Build the IREE runtime on the macOS host for iOS devices and the
+  simulator.  We can then run the IREE module on the simulator.
+  
+## Prerequisites
+
+### Install Xcode and iOS SDK
+
+For cross-compilation, you need Xcode. It comes with the SDKs for iOS
+devices and the simulator, as well as the `simctl` tool for
+controlling the simulator from the command line.
```

Comment:
Can you include a link here for downloading Xcode? https://developer.apple.com/xcode/ looks like a reasonable page to link to (it shows information about the program and includes download links)

Reply:
The App Store is the best place to get any app for macOS. We only download from the link above if we need more than one version on the host. In that case, we will need to use the "xcode-select" command to choose the default.

Reply:
I'd prefer linking to https://developer.apple.com/xcode/ instead of https://apps.apple.com/us/app/xcode/id497799835 (the developer page looks nicer, includes more information, and links to the app store)

Some enterprise users also download applications from private sources instead of the App Store, so linking to a general information page first might avoid some confusion for them.

Reply:
Added the link https://developer.apple.com/xcode/.